clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -507,7 +513,24 @@ scheduler_tmpl_t<RecordType, ReaderType>::init(
                                   // TODO i#5843: Add more flags for other options.
                                   : spec_type_t::LAST_FROM_TRACE,
                               verbosity_);
+        if (options_.record_ostream != nullptr) {
+            sched_type_t::stream_status_t status = record_schedule_segment(
+                i, schedule_record_t::VERSION, schedule_record_t::VERSION_CURRENT, i, 0);
```

Comment:
Here we're seeing `input` to `schedule_record_t::VERSION_CURRENT`?

Reply:
Since input_ordinal_t and version are both ints we can't overload unfortunately.  Added a comment to record_schedule_segment's declaration.